guardrails/validator_base.py

```diff
@@ -448,10 +489,49 @@ def __init__(
             self.rail_alias in validators_registry
         ), f"Validator {self.__class__.__name__} is not registered. "
 
+    def chunking_function(self, chunk: str):
```

Comment:
Is this always set based on defining the chunking_function? Should either document this or we should set in the init

Reply:
Nevermind, maybe doing this in the init is overkill. I think this is ok. The function of the chunking strategy in the validator should not be determined by the user imho

Reply:
This was originally set in the init but the typing gets weird when we set it up that way.  When it's defined as a class method like this, other validators can simply override it.